### ISSUE TITLE
LOKI_LISTEN_PORT not replaced with default in nginx config

### ISSUE THREAD

**Issue by radarsymphony** [nfrastack/container-loki#2]
### Summary

The  upstream loki server port is not currently being replaced/updated fully. The place-holder [`{{LOKI_LISTEN_PORT}}`](https://github.com/tiredofit/docker-loki/blob/f6093c9b82e020f8b56f4a0bbfe264461ba36b4b/install/etc/nginx/sites.available/loki.conf#L2) is being removed by [`update_template`](https://github.com/tiredofit/docker-alpine/blob/94ea7941e7a65cc216ff6887bd623ab585599815/install/assets/functions/00-container#L1116) but it looks like that variable is empty or doesn't exist because the resulting nginx config is missing the default port.

### Steps to reproduce

1. Spin up the `tiredofit/loki:latest`
2. Watch docker-compose logs
3. See error `nginx: [emerg] invalid port in upstream "127.0.0.1:" in /etc/nginx/sites.enabled/loki.conf:2`

### What is the expected *correct* behavior?

The default loki port should be inserted into the nginx template if none is supplied in the docker-compose.yml. 

### Relevant logs and/or screenshots

N/A

### Environment
- Image version / tag:  tiredofit/loki:latest

```
            "Labels": {
                "maintainer": "Dave Conroy (github.com/tiredofit)",
                "tiredofit.base.git_branch": "master",
                "tiredofit.base.git_changelog_version": "7.7.25",
                "tiredofit.base.git_commit": "64d98042775ef09a30d785ab2cf1639868a0fb60",
                "tiredofit.base.git_committed_by": "tiredofit",
                "tiredofit.base.git_repository": "https://github.com/tiredofit/docker-alpine",
                "tiredofit.base.image_build_date": "2022-10-05 00:39:09",
                "tiredofit.base.runtime_version": "3.16",
                "tiredofit.image.git_branch": "main",
                "tiredofit.image.git_commit": "f6093c9b82e020f8b56f4a0bbfe264461ba36b4b",
                "tiredofit.image.git_committed_by": "tiredofit",
                "tiredofit.image.git_repository": "https://github.com/tiredofit/docker-loki",
                "tiredofit.image.image_build_date": "2022-10-05 18:01:31",
                "tiredofit.loki.git_changelog_version": "0.0.17",
                "tiredofit.nginx.git_branch": "master",
                "tiredofit.nginx.git_changelog_version": "6.2.5",
                "tiredofit.nginx.git_commit": "d04f6e3107bd3eb1b76493227c3c634ee83a016d",
                "tiredofit.nginx.git_committed_by": "tiredofit",
                "tiredofit.nginx.git_repository": "https://github.com/tiredofit/docker-nginx",
                "tiredofit.nginx.image_build_date": "2022-10-05 17:29:37"
```

- Host OS: Arch Linux

<details>
<summary>Any logs | docker-compose.yml</summary>

```
loki-app  | 2022-10-28.07:21:23 [NOTICE] ** [nginx] Creating sample index.html
loki-app  | 2022-10-28.07:21:23 [STARTING] ** [nginx] [1] Starting nginx 1.23.1
loki-app  | nginx: [emerg] invalid port in upstream "127.0.0.1:" in /etc/nginx/sites.enabled/loki.conf:2
loki-app  | 2022-10-28.07:21:23 [STARTING] ** [monitoring] [1] Starting Zabbix Agent (modern) 6.2.3
loki-app  | 2022-10-28.07:21:23 [STARTING] ** [loki] [1] Starting loki v2.6.1
loki-app  | 2022-10-28.07:21:23 [STARTING] ** [scheduling] [1] Starting cron
```

</details>

    
### Possible fixes

The nginx template is being updated at [this line](https://github.com/tiredofit/docker-loki/blob/f6093c9b82e020f8b56f4a0bbfe264461ba36b4b/install/etc/cont-init.d/20-loki#L82) in the code. The variable name passed in is `LOKI_LISTEN_PORT` but the variable name for the default port in the **defaults** file is `LISTEN_PORT` at [this line](https://github.com/tiredofit/docker-loki/blob/f6093c9b82e020f8b56f4a0bbfe264461ba36b4b/install/assets/defaults/20-loki#L22). Should these variable names be the same?

EDIT:

Adding the following to the docker-compose.yml provides a temporary workaround/fix:

```
   environment:
      # Temporary test/fix for https://github.com/tiredofit/docker-loki/issues/2
      - LOKI_LISTEN_PORT=3100
```

**Comment by tiredofit** [comment]
Thanks - Resolved in `0.0.18` - Going to stick with the `LISTEN_PORT` variable, Nginx will configure now :)

**Comment by radarsymphony** [comment]
`0.0.18` is working for me. Thank you!
